data/filters/youtube-shorts.yaml

```diff
@@ -7,11 +7,6 @@ template: |
   www.youtube.com##ytd-mini-guide-renderer a.yt-simple-endpoint[title="Shorts"]:upward(ytd-mini-guide-entry-renderer)
   {{! Homepage shelf }}
   www.youtube.com##ytd-browse #dismissible ytd-rich-grid-slim-media[is-short]:upward(ytd-rich-section-renderer)
-  {{! Old style with duration, still used in homepage? }}
-  www.youtube.com##ytd-browse ytd-grid-video-renderer:has(span.ytd-thumbnail-overlay-time-status-renderer:has-text(/\s(0:\d\d|1:0\d)\s/))
-  www.youtube.com##ytd-browse ytd-rich-item-renderer:has(span.ytd-thumbnail-overlay-time-status-renderer:has-text(/\s(0:\d\d|1:0\d)\s/))
-  www.youtube.com##ytd-search ytd-video-renderer:has(span.ytd-thumbnail-overlay-time-status-renderer:has-text(/\s(0:\d\d|1:0\d)\s/))
-  www.youtube.com##ytd-watch-next-secondary-results-renderer ytd-compact-video-renderer:has(span.ytd-thumbnail-overlay-time-status-renderer:has-text(/\s(0:\d\d|1:0\d)\s/))
```

Comment:
Good call on this one, it was on my todo-list :bow: 

When I checked the homepage last weekend:

- most shorts are now in the dedicated shelf, that we have a rule for
- I still found one short amongst the other videos, and the overlay did show a duration instead of the logo.

I'll need to double-check that, but if that's true, we might want to keep the first rule only (and maybe make it more selective with `ytd-browse[page-subtype="home"]`. What do you think?

Reply:
If you have double-checked it and you still find the overlay showing a duration instead of the logo,
making it more selective with `ytd-browse[page-subtype="home"]` sounds like a good idea.